runtime/avn/src/lib.rs

```diff
@@ -946,9 +957,14 @@ impl_runtime_apis! {
     }
 
     impl pallet_eth_bridge_runtime_api::EthEventHandlerApi<Block, AccountId> for Runtime {
-        fn query_current_author() -> Option<AccountId>{
-            Avn::get_validator_for_current_node()
-                .map(|validator| validator.account_id)
+        fn query_authors() ->Option<Vec<[u8; 32]>> {
```

Comment:
I think we should rename this function to make it explicit that its returning the signing keys of the authors, not the authors themselves. The big difference here is that the authors (author address) cannot be changed but the signing key can be rotated. And unsigned transactions are signed using the signing key. So i would suggest renaming it to `query_author_signing_keys`